src/libpgmoneta/memory.c

```diff
@@ -122,3 +122,40 @@ pgmoneta_memory_destroy(void)
    data = NULL;
    message = NULL;
 }
+
+void*
+pgmoneta_memory_dynamic_create(size_t* size)
+{
+   *size = 0;
+
```

Comment:
All it does is set the size? What about the memory it created? @jesperpedersen

Reply:
Yeah, we will start with `NULL`